src/server/graphql/rootQuery.js

```diff
@@ -1,10 +1,12 @@
 import {GraphQLObjectType} from 'graphql';
 import User from 'server/graphql/types/User';
 import {getUserId} from 'server/utils/authorization';
+import activeProUserCount from 'server/graphql/queries/activeProUserCount';
 
 export default new GraphQLObjectType({
   name: 'Query',
   fields: () => ({
+    activeProUserCount,
```

Comment:
I was wondering what the pattern was going to look like for adding this stupid little query. That's what was keeping me from doing it myself.

Reply:
@jordanh totally, this is our very first query that isn't under the viewer object! i mean i suppose we still could stick it under there, but i kinda like it on its own because it says "hey, these are the things that a normal viewer doesn't care about/have access to".